biz.aQute.bnd/src/aQute/bnd/main/XRefCommand.java

```diff
@@ -57,6 +57,8 @@ interface xrefOptions extends Options {
 		@Description("Output list of package/class names that have been referred to")
 		String referrredTo();
 
+		@Description("Include java.* packages")
+		boolean java();
```

Comment:
Don't you need to evaluate that flag somewhere in https://github.com/bndtools/bnd/blob/d56166137d9b1e82381d205b1837998cd3fad87e/biz.aQute.bnd/src/aQute/bnd/main/XRefCommand.java#L73?